test/01_artist_token.js

```diff
@@ -109,7 +109,7 @@ contract("ArtistToken", ([lightstreams, hatcher1, hatcher2, lateInvestor]) => {
               it("Should have set the locked internal tokens for the hatcher", async function() {
                 let initialContributions = await artistToken.initialContributions(hatcher1);
                 let lockedInternal = initialContributions.lockedInternal;
-                assert.equal(lockedInternal.toString(), (INSUFFICIENT_AMOUNT_TO_RAISE_WEI * P0).toString());
+                assert.equal(lockedInternal.toString(), (INSUFFICIENT_AMOUNT_TO_RAISE_WEI/ 4).toString());
```

Comment:
Do you think you could calculate this number on the fly instead of hard-coding a 4 here to represent the 25 000 ppm p0 price?

Reply:
This was to confirm to me that the P0 ratio of 0.25 causes the result to divide by 4. But, yes, agreed this could be calculated on the fly as P0/DENOMINATOR_PPM